public/locales/en-US.json

```diff
@@ -29,9 +29,9 @@
     "cardHeading_Satellite Capsule": "CPU socket usage",
     "cardHeading_Satellite Server": "CPU socket usage",
     "cardHeadingDescription": "...",
-    "cardHeadingDescription_Instance-hours": "Instance hours usage in gibibyte hours",
-    "cardHeadingDescription_Storage-gibibytes": "Data storage usage in gibibyte hours",
-    "cardHeadingDescription_Transfer-gibibytes": "Gibibyte hour usage per instance",
+    "cardHeadingDescription_Instance-hours": "Instance hours usage in binary gigabyte hours",
+    "cardHeadingDescription_Storage-gibibytes": "Data storage usage in binary gigabyte hours",
```

Comment:
@jlprevatt confirm this changes towards
```
"cardHeadingDescription_Storage-gibibytes": "Data storage usage in binary gigabytes"
```

Reply:
This one is actually correct in the original. Storage is counted as amount over time (with that fancy "sum over time" sampling formula), so:
"Data storage usage in binary gigabyte hours"